__tests__/integration.test.js

```diff
@@ -84,86 +84,140 @@ describe("getArticleByID", () => {
+        });
+      });
+  });
+  it("should return the object in the correct order, with correct length and order", () => {
     return request(app)
-    .get('/api/articles')
-    .then((res) => {
-      const {articles} = res.body
-      expect(articles).toHaveLength(13);
-      expect(articles).toBeSortedBy('created_at', {descending: true});
-    })
+      .get("/api/articles")
+      .then((res) => {
+        const { articles } = res.body;
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
   });
 });
 
-describe('getCommentsByID', () => {
-it('should correctly return array of objects with a status 200 code', () => {
-  return request(app)
-  .get('/api/articles/1/comments')
-  .expect(200)
-  .then((res) => {
-    const {comments} = res.body
-    expect(comments).toHaveLength(11);
-    comments.forEach((comment) => {
-      expect(typeof comment.comment_id).toBe('number');
-      expect(typeof comment.votes).toBe('number');
-      expect(typeof comment.created_at).toBe('string');
-      expect(typeof comment.author).toBe('string');
-      expect(typeof comment.body).toBe('string');
-      expect(typeof comment.article_id).toBe('number');
-    });
-  })
-});
-it('should return a 404 whenn passed an article id that does not exist', () => {
-  return request(app)
-  .get('/api/articles/999/comments')
-  .expect(404)
-  .then((res) => {
-    expect(res.body.error).toBe('Not found');
-  })
-});
-it('should return 400 when passed an invalid type of article id', () => {
-  return request(app)
-  .get('/api/articles/dog/comments')
-  .expect(400)
-  .then((res) => {
-    expect(res.body.msg).toBe('Bad request');
-  })
-})
-it('should return object with correct  order', () => {
-  return request(app)
-  .get('/api/articles/1/comments')
-  .then((res) => {
-    const {comments} = res.body
-    expect(comments).toBeSortedBy('created_at', {descending: true});
-  })
+describe("getCommentsByID", () => {
+  it("should correctly return array of objects with a status 200 code", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        const { comments } = res.body;
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+        });
+      });
+  });
+  it("should return a 404 whenn passed an article id that does not exist", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.error).toBe("Not found");
```

Comment:
we should be consistent on how to access the error messages - it looks like they are either available on res.body.error OR res.body.msg. let's choose one and stick with one so it's not confusing for the user